packages/overlays/package.json

```diff
@@ -37,6 +37,8 @@
     "popper.js": "^1.15.0"
   },
   "devDependencies": {
+    "@lion/button": "^0.3.32",
```

Comment:
circle dependencies everywhere...
we should move demos that require other stuff into a separate package... 

currently, every time you change one package => every package get's released because of dev dependencies...

Reply:
fyi: not a real block - but it's something we should address at some point
=> opened an issue https://github.com/ing-bank/lion/issues/414